cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""This module tests Nuage EMS events."""
+import pytest
+from wrapanapi.utils.random import random_name
+
+from cfme.networks.provider.nuage import NuageProvider
+from cfme.utils.log import logger
+
+
+pytestmark = [
+    pytest.mark.provider([NuageProvider])
+]
+
+
+@pytest.yield_fixture
```

Comment:
yield_fixture is deprecated.  its functionality has been moved to pytest.fixture. please use it instead.